jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -361,4 +361,46 @@ class EndpointK @JvmOverloads constructor(
             conferenceStats.dtlsFailedEndpoints.incrementAndGet()
         }
     }
+
+    override fun expire() {
+        if (super.isExpired()) {
+            return
+        }
+        super.expire()
+
+        try {
+            val channelShimsCopy = channelShims.toSet()
+            channelShims.clear()
+            channelShimsCopy.forEach { channelShim ->
+                if (!channelShim.isExpired) {
+                    channelShim.expire = 0
+                }
+            }
+            updateStatsOnExpire()
+            transceiver.stop()
+            logger.cdebug { transceiver.getNodeStats().prettyPrint(0) }
+            logger.cdebug { bitrateController.debugState.toJSONString() }
+            logger.cdebug { iceTransport.getDebugState().toJSONString() }
+            logger.cdebug { dtlsTransport.getDebugState().toJSONString() }
+
+            logger.info("Spent ${bitrateController.getTotalOversendingTime().seconds} seconds oversending")
```

Comment:
Could this use `cinfo`, for parallel to the above logs?

Reply:
There isn't a `cinfo` defined since we don't ever worry about guarding `INFO` logs with checks.

Reply:
??? https://github.com/jitsi/jitsi-utils/blob/5d25e9b4a5c19f52dba48fdc3e7804a68a1eaf29/src/main/kotlin/org/jitsi/utils/logging2/LoggerExtensions.kt#L58

Reply:
Ah, weird...it didn't auto complete for me.  Still we haven't really used `cinfo` anywhere, I think I'd rather leave that for a single, focused effort to make all info logs consistent if we want that.

Reply:
Ok.

`getTotalOversendingTime()` doesn't look to be too expensive, either, even if `INFO` logs were turned off.